docs/pages/identity-governance/access-request-plugins/datadog-hosted.mdx

```diff
@@ -141,9 +141,8 @@ to create a new Application key. Copy the Application key to paste in a later st
 ## Step 4/6. Enroll the Datadog Incident Management plugin
 
 At this point, you're now ready to enroll the Datadog Incident Management plugin.
-Navigate to **Access Management -> Enroll New Integration -> Datadog**.
 
-![Select enrollment](../../../img/enterprise/plugins/datadog/select-enrollment.png)
+(!docs/pages/includes/plugins/enroll.mdx name="the Datadog Incident Management"!)
```

Comment:
I would also use a horizontal layout for the image here. On my monitor, with 100 percent browser zoom, the image takes up most of the viewport.